target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -1118,7 +1223,8 @@ TARGET_DEVICES += CPVA502PLUS
 
 define Device/CPA-ZNTE60T
   $(Device/bcm63xx)
-  DEVICE_TITLE := Telsey CPVA642-type (CPA-ZNTE60T)
+  DEVICE_VENDOR := Telsey
+  DEVICE_MODEL := CPVA642-type (CPA-ZNTE60T)
```

Comment:
CPA-ZNTE60T might be an alternate name?

Reply:
CPA-ZNTE60T seems to be the box label. Which corresponds to a hardware revision of the CPVA64* device: https://repository.ilpuntotecnico.com/files/merlino60/telsey%20fastweb/documenti%20vari/MU642-TLYIT10.pdf

Looks like they smoke too much.

Reply:
Not sure what to do with that. Could also live with keeping the current version.

Reply:
No idea, let's keep it.